fe/lib/components/LocationSuggest/LocationSuggestInput.tsx

```diff
@@ -153,6 +179,45 @@ const LocationSuggestInput = ({
           value={locationSuggest}
         />
       </Column>
+      <Column width="content">
+        <Button
+          icon={<IconLanguage />}
+          onClick={() => setShowMapDialog(true)}
+          tone={tone === 'critical' ? tone : undefined}
+          variant="soft"
+        >
+          Select on map
+        </Button>
+        <Dialog
+          id="map-dialog"
```

Comment:
I think we should namespace this at minimum to avoid conflicts in the DOM:

```suggestion
          id="seek-location-suggest-map-dialog"
```

Though long term we should move towards [useId](https://reactjs.org/docs/hooks-reference.html#useid)